main.go

```diff
@@ -247,7 +246,12 @@ func sendToWriter(ingestionType string, msgBody *strings.Reader, uuid string, ur
 
 	request, err := http.NewRequest("PUT", reqURL, msgBody)
 	request.ContentLength = -1
-	resp, err := httpClient.Do(request)
+
+	client1 := pester.New()
```

Comment:
Can we reuse clients, like we did with the simple http client? Also, are there several clients? Why `client1`?